apps/namadillo/src/App/AccountOverview/UnshieldedAssetTable.tsx

```diff
@@ -144,6 +144,7 @@ const TransparentTokensTable = ({
         Tokens
       </div>
       <TableWithPaginator
+        containerClassName="h-screen"
```

Comment:
It looks like it's a bit bigger than necessary. Maybe you will need to add a bunch of `flex:1` to fix this 😬 


https://github.com/user-attachments/assets/b63c2b0c-f2fb-4a01-910a-f4763e5d1580

Reply:
Yeah the former alternative was too small unfortunately 🫠, this will all be coming in the cleanup PR with Chris' other suggestions